Orange/widgets/data/owpythonscript.py

```diff
@@ -480,9 +479,8 @@ def __init__(self):
 
         self.controlBox.layout().addWidget(w)
 
-        auto = gui.auto_commit(self.controlArea, self, "auto_execute", "Run",
-                               checkbox_label="Autorun on new data")
-        self.execute_button, self.autobox = auto.button, auto.checkbox
+        self.execute_button = QPushButton(self, text='Run', clicked=self.commit)
+        self.controlArea.layout().addWidget(self.execute_button)
```

Comment:
We prefer using `gui.button` in widgets. Not much to gain here, just consistency.